### PR TITLE
Feature/support actix web

### PR DESCRIPTION
**Pull request by biryukovmaxim**
#205 
implementation of Service for actix-web using [configure method](https://actix.rs/docs/application)
[Example](https://github.com/biryukovmaxim/shuttle-examples/tree/feature/actix-web-example/actix-web/hello-world)

cargo check/build without error, but when I try to deploy locally the example, deploy get crashed providing these logs:

<details>
  <summary>Logs</summary>
 `
[2m2022-11-27T16:57:11.637728131Z[0m [38;5;10m INFO[39m [38;5;12m[1mEntering queued state[0m

[2m2022-11-27T16:57:11.638499343Z[0m [38;5;10m INFO[39m [38;5;12m[1mEntering building state[0m
[2m2022-11-27T16:57:11.942690023Z[0m [38;5;10m INFO[39m     Updating crates.io index
[2m2022-11-27T16:57:20.806397961Z[0m [38;5;10m INFO[39m warning: Patch `shuttle-aws-rds v0.7.2 (/usr/src/shuttle/resources/aws-rds)` was not used in the crate graph.
[2m2022-11-27T16:57:20.806455747Z[0m [38;5;10m INFO[39m Patch `shuttle-persist v0.7.2 (/usr/src/shuttle/resources/persist)` was not used in the crate graph.
[2m2022-11-27T16:57:20.811746749Z[0m [38;5;10m INFO[39m Patch `shuttle-secrets v0.7.2 (/usr/src/shuttle/resources/secrets)` was not used in the crate graph.
[2m2022-11-27T16:57:20.816806560Z[0m [38;5;10m INFO[39m Patch `shuttle-shared-db v0.7.2 (/usr/src/shuttle/resources/shared-db)` was not used in the crate graph.
[2m2022-11-27T16:57:20.821676231Z[0m [38;5;10m INFO[39m Check that the patched package version and available features are compatible
[2m2022-11-27T16:57:20.826547901Z[0m [38;5;10m INFO[39m with the dependency requirements. If the patch has a different version from
[2m2022-11-27T16:57:20.831885817Z[0m [38;5;10m INFO[39m what is locked in the Cargo.lock file, run `cargo update` to use the new
[2m2022-11-27T16:57:20.837278048Z[0m [38;5;10m INFO[39m version. This may also occur with an optional dependency that is not enabled.
[2m2022-11-27T16:57:46.372509914Z[0m [38;5;10m INFO[39m    Compiling zstd-sys v2.0.3+zstd.1.5.2
[2m2022-11-27T16:57:46.560632521Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:218:21
    |
218 |     pub lowerBound: ::core::ffi::c_int,
    |                     ^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.565495975Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:219:21
    |
219 |     pub upperBound: ::core::ffi::c_int,
    |                     ^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.570742589Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zdict_std.rs:51:27
   |
51 |     pub compressionLevel: ::core::ffi::c_int,
   |                           ^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.575906718Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zdict_std.rs:52:28
   |
52 |     pub notificationLevel: ::core::ffi::c_uint,
   |                            ^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.581057989Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zdict_std.rs:53:17
   |
53 |     pub dictID: ::core::ffi::c_uint,
   |                 ^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.601148984Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:53:36
   |
53 |     pub fn ZSTD_versionNumber() -> ::core::ffi::c_uint;
   |                                    ^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.610123963Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:57:43
   |
57 |     pub fn ZSTD_versionString() -> *const ::core::ffi::c_char;
   |                                           ^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.616819431Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:66:27
   |
66 |         compressionLevel: ::core::ffi::c_int,
   |                           ^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.623070379Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:82:10
   |
82 |     ) -> ::core::ffi::c_ulonglong;
   |          ^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.628803332Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:89:10
   |
89 |     ) -> ::core::ffi::c_ulonglong;
   |          ^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.634203539Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:102:41
    |
102 |     pub fn ZSTD_isError(code: usize) -> ::core::ffi::c_uint;
    |                                         ^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.639573315Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:105:53
    |
105 |     pub fn ZSTD_getErrorName(code: usize) -> *const ::core::ffi::c_char;
    |                                                     ^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.644821276Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:108:32
    |
108 |     pub fn ZSTD_minCLevel() -> ::core::ffi::c_int;
    |                                ^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.650085570Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:111:32
    |
111 |     pub fn ZSTD_maxCLevel() -> ::core::ffi::c_int;
    |                                ^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.655443069Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:114:36
    |
114 |     pub fn ZSTD_defaultCLevel() -> ::core::ffi::c_int;
    |                                    ^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.661930465Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:137:27
    |
137 |         compressionLevel: ::core::ffi::c_int,
    |                           ^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.681132389Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:230:16
    |
230 |         value: ::core::ffi::c_int,
    |                ^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.690501523Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:237:25
    |
237 |         pledgedSrcSize: ::core::ffi::c_ulonglong,
    |                         ^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.697097729Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:283:16
    |
283 |         value: ::core::ffi::c_int,
    |                ^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.703152686Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:350:27
    |
350 |         compressionLevel: ::core::ffi::c_int,
    |                           ^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.709034150Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:408:27
    |
408 |         compressionLevel: ::core::ffi::c_int,
    |                           ^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.714705164Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:435:27
    |
435 |         compressionLevel: ::core::ffi::c_int,
    |                           ^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.720161582Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:486:10
    |
486 |     ) -> ::core::ffi::c_uint;
    |          ^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.737224686Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:492:10
    |
492 |     ) -> ::core::ffi::c_uint;
    |          ^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.741749379Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:498:10
    |
498 |     ) -> ::core::ffi::c_uint;
    |          ^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.747551978Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zstd_std.rs:505:10
    |
505 |     ) -> ::core::ffi::c_uint;
    |          ^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.753276538Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zdict_std.rs:45:20
   |
45 |         nbSamples: ::core::ffi::c_uint,
   |                    ^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.758725255Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zdict_std.rs:64:20
   |
64 |         nbSamples: ::core::ffi::c_uint,
   |                    ^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.764300068Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zdict_std.rs:72:10
   |
72 |     ) -> ::core::ffi::c_uint;
   |          ^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.769786361Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zdict_std.rs:81:47
   |
81 |     pub fn ZDICT_isError(errorCode: usize) -> ::core::ffi::c_uint;
   |                                               ^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.787381341Z[0m [38;5;12mDEBUG[39m error[E0658]: use of unstable library feature 'core_ffi_c'
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-2.0.3+zstd.1.5.2/src/bindings_zdict_std.rs:84:59
   |
84 |     pub fn ZDICT_getErrorName(errorCode: usize) -> *const ::core::ffi::c_char;
   |                                                           ^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #94501 <https://github.com/rust-lang/rust/issues/94501> for more information

[2m2022-11-27T16:57:46.792197365Z[0m [38;5;12mDEBUG[39m error: aborting due to 31 previous errors

[2m2022-11-27T16:57:46.798011748Z[0m [38;5;12mDEBUG[39m For more information about this error, try `rustc --explain E0658`.

[2m2022-11-27T16:57:46.814472784Z[0m [38;5;10m INFO[39m error: could not compile `zstd-sys` due to 32 previous errors
[2m2022-11-27T16:57:46.819318702Z[0m [38;5;10m INFO[39m warning: build failed, waiting for other jobs to finish...

[2m2022-11-27T16:57:46.948204213Z[0m [38;5;10m INFO[39m [38;5;12m[1mEntering crashed state[0m
[2m2022-11-27T16:57:46.948441338Z[0m [38;5;9mERROR[39m {error="Build error: 1 job failed"} [2mshuttle_deployer::deployment::queue:[0m service build encountered an error
`

  
</details>
I have no idea how to fix this error

### CODE REVIEW

File: service/src/lib.rs [hunk 1]
```diff
@@ -544,6 +546,22 @@ impl Service for sync_wrapper::SyncWrapper<axum::Router> {
...
 }
 
+#[cfg(feature = "web-actix-web")]
+#[async_trait]
+impl<F> Service for F
+where F: FnOnce(&mut actix_web::web::ServiceConfig) + Sync + Send + Copy + Clone + 'static {
+    async fn bind(self: Box<Self>, addr: SocketAddr) -> Result<(), Error> {
+        let srv = actix_web::HttpServer::new(move || actix_web::App::new().configure(*self))
+            .bind(addr)?
+            .run();
```
Comment (chesedo) [inline service/src/lib.rs:556]: Oh, nice!! Quite a few of us tried an implementation before but could not get it working.

**Review by chesedo — COMMENTED** [review]
This is great, thanks @biryukovmaxim!

Can you also add a simple hello world example to the [examples repo](https://github.com/shuttle-hq/examples) and update the `cargo-shuttle` integrations test and the `e2e` tests with the example added.

As for the error, does a `cargo shuttle run` work?

**Comment by oddgrd** [comment]
To add to what chesedo said, thanks for the PR and good job! This looks promising. Regarding that error, it seems the zstd crate bumped their minimum supported rust version to 1.64, and zstd may be a dependency of actix. Currently our containers are pinned to 1.63, but we're looking into upgrading. In the meantime you could try testing it out with `cargo shuttle run` like chesedo suggested.

**Comment by oddgrd** [comment]
On second thought, adding this to `shuttle-service` Cargo.toml should also work: `zstd-sys = "=2.0.1+zstd.1.5.2"`.

File: service/src/lib.rs (hunk 1 above)
Comment (biryukovmaxim) [reply service/src/lib.rs:556]: @chesedo Yep, `cargo shuttle run` works! I've implemented the example, but haven't created a PR yet.

**Comment by biryukovmaxim** [comment]
> This is great, thanks @biryukovmaxim!
> 
> Can you also add a simple hello world example to the [examples repo](https://github.com/shuttle-hq/examples) and update the `cargo-shuttle` integrations test and the `e2e` tests with the example added.
> 
> As for the error, does a `cargo shuttle run` work?

[PR to examples](https://github.com/shuttle-hq/examples/pull/3)

**Comment by biryukovmaxim** [comment]
> On second thought, adding this to `shuttle-service` Cargo.toml should also work: `zstd-sys = "=2.0.1+zstd.1.5.2"`, this pins it to a version compatible with rust 1.63.

I added dependency, now it works!
JUST FYI Unfortunately it can't be compiled with nightly(1.67)

**Comment by biryukovmaxim** [comment]
> On second thought, adding this to `shuttle-service` Cargo.toml should also work: `zstd-sys = "=2.0.1+zstd.1.5.2"`, this pins it to a version compatible with rust 1.63.

I added dependency, now it works!
JUST FYI Unfortunately it can't be compiled with nightly(1.67)

**Comment by oddgrd** [comment]
Nice! I just saw that they released a new version of zstd-sys now, that fixes the accidental MSRV increase. Meaning you should be able to remove it from `shuttle-service`, and then update `Cargo.lock`.

The reason it failed in CI now is probably because it was missing the equals sign in `Cargo.toml`: `zstd-sys = "=2.0.1+zstd.1.5.2"`, which is what forces it to be that exact version. But that shouldn't be required anymore, now it should default to zstd-sys = 2.0.4 which has a lower msrv.

**Review by chesedo — COMMENTED** [review]
Just one more test is missing for the [run integrations](https://github.com/shuttle-hq/shuttle/blob/main/cargo-shuttle/tests/integration/run.rs), then I'm happy to approve this :smile:

**Comment by biryukovmaxim** [comment]
> Just one more test is missing for the [run integrations](https://github.com/shuttle-hq/shuttle/blob/main/cargo-shuttle/tests/integration/run.rs), then I'm happy to approve this smile

Is it possible to add this test without updating examples-repo first?

**Comment by oddgrd** [comment]
For local runs you can `cd` into examples and checkout your PR branch. 

For CI you can do this and we'll remove it when we're ready to merge both this and the new example:

```
# in .gitmodules

[submodule "examples"]
	path = examples
	url = git@github.com:biryukovmaxim/shuttle-examples.git
	branch = feature/actix-web-example
```

**Comment by biryukovmaxim** [comment]
done!

**Review by oddgrd — APPROVED** [review]
Thank you @biryukovmaxim, we are so happy with this PR and to finally have actix support! :partying_face:

**Review by chesedo — APPROVED** [review]

**Comment by biryukovmaxim** [comment]
> @biryukovmaxim if you ever write a blog post or something to explain how you figured this out, I would love to read it smile
@chesedo  Actually, It's not my merit. I tryed to implement it with `App<AppEntry>`, then I faced with [issue](https://github.com/actix/actix-web/issues/780).
So I checked every issue from [this comment](https://github.com/actix/actix-web/issues/780#issuecomment-894704144) and found that  [answer](https://github.com/actix/actix-web/issues/2039#issuecomment-789045650)

**Pull request merged**